src/allmydata/test/test_configutil.py

```diff
@@ -20,10 +32,22 @@ def setUp(self):
             lambda section_name, item_name: (section_name, item_name) == ("node", "valid"),
         )
 
+    def create_tahoe_cfg(self, cfg):
+        d = self.mktemp()
+        os.mkdir(d)
+        fname = os.path.join(d, 'tahoe.cfg')
+        with open(fname, "w") as f:
+            f.write(cfg)
+        return fname
```

Comment:
Just making a note, nothing to act upon here: 

Creating node configuration is a pattern repeated fairly often in the test suite.  I guess it could be moved somewhere else (`test.common` maybe), but that is obviously not in the scope of this PR.  It is quite useful that usage of this pattern in this module is collected in one place now!